### PR TITLE
Feature/build infrastructure and docker image creation

### PR DESCRIPTION
**Pull request by jmaslibre**
This PR includes a basic java program as a placeholder and creates and pushes the docker image to dockerhub, using travis-ci.

Here is the branch job: https://travis-ci.org/performance-data-collection/algorithms

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM oracle/openjdk
```
Comment (wvides) [inline Dockerfile]: Shouldn't this be referencing a specific tag? to ensure our performance tests always perform the same?

**Review by wvides — COMMENTED** [review]
Everything looks awesome! my only concern is about the tag of the docker image you are currently referencing at the Dockefile :)

File: Dockerfile (hunk 1 above)
Comment (jmaslibre) [reply Dockerfile]: Makes sense, I will fix it.

Comment (jmaslibre) [reply Dockerfile]: Fixed.

**Review by wvides — APPROVED** [review]

**Pull request merged**